build.sbt

```diff
@@ -100,7 +100,9 @@ lazy val root = Project("ctd-omega-editorial-frontend", file("."))
       "-Ywarn-unused:params", // Warn if a value parameter is unused.
       "-Ywarn-unused:patvars", // Warn if a variable bound in a pattern is unused.
       "-Ywarn-unused:privates", // Warn if a private member is unused.
-      "-Ywarn-value-discard" // Warn when non-Unit expression results are unused.  //
+      "-Ywarn-value-discard", // Warn when non-Unit expression results are unused.  //
+      "-target:jvm-1.8",
```

Comment:
I think this should now say `-release:8` as `-target` is deprecated - see "Targeting a version of the JVM" at https://docs.scala-lang.org/overviews/compiler-options/index.html